apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx

```diff
@@ -29,7 +30,7 @@ export default class ModerationNotificationList extends Component {
 
     this.state = {
       moderationComments: [],
-      selectedFilters: { isRead: 'False', hasReports: 'All', ordering: '-num_reports' },
+      selectedFilters: { isRead: 'False', reportedBy: 'off', ordering: '-num_reports' },
```

Comment:
I think the "Most reported" filter still just depends on the field `num_reports`, i.e. is unaffected by the re-naming of `hasReports`. Or am I missing something?

Reply:
indeed, is the annotation `num_reposts` you changed in the userdashboard.api.py, and now the queryset returns comments based on ai or user filtering, instead of boolean has reports or not.

Reply:
let's talk about it with a screen share, I feel that I can't follow you 100%